consensus/spos/bls/subroundEndRound.go

```diff
@@ -626,9 +750,6 @@ func (sr *subroundEndRound) createAndBroadcastInvalidSigners(invalidSigners []by
 }
 
 func (sr *subroundEndRound) doEndRoundJobByParticipant(cnsDta *consensus.Message) bool {
-	sr.mutProcessingEndRound.Lock()
```

Comment:
As far as I remember, it should be one level up, in case self receives some final info, to not start aggregating signatures on its own. There was some concurrency problem while testing.